mockup/patterns/relateditems/pattern.js

```diff
@@ -99,27 +99,30 @@ define([
       homeText: _t('home'),
       folderTypes: ['Folder'],
       selectableTypes: null, // null means everything is selectable, otherwise a list of strings to match types that are selectable
-      attributes: ['UID', 'Title', 'portal_type', 'path', 'getIcon'],
+      attributes: ['UID', 'Title', 'portal_type', 'path','getURL', 'getIcon','contenttype_class','is_folderish','review_state'],
```

Comment:
contenttype_class: the css class to be rendered for a contentttype.

the existing code passes over portal_type from vocabulary, then I have seen approaches like portal.type.to:LowerCase() to render the css class. This works sometimes but not for 'News Item' or other (custom dx) types. Of course you could construct some js solution, but the easiest and cleanest way is to use 'contenttype-'+normalizeString(portal_type) which guarentees conformity even when someone alters the implementation of normalizeString.  see ...
´´´´
      if key == 'contenttype_class':
+                        pt = getattr(vocab_item, 'portal_type', None)
+                        val = 'contenttype-%s' % normalizeString(pt)
                     item[key] = val
````
in https://github.com/plone/plone.app.content/pull/57 :   plone/app/content/browser/vocabulary.py 

If this is a bad solution, or I am totally wrong, or you think we should just rename contenttype_class ( contenttype_css,
 contenttype_css-class, or whatever) pleas let me know.